plonk-core/src/proof_system/prover.rs

```diff
@@ -247,6 +368,29 @@ where
         // Add permutation polynomial commitment to transcript.
         transcript.append(b"z", z_poly_commit[0].commitment());
 
+        // Compute mega permutation polynomial.
+        // Compute lookup permutation poly
+        let z_2_poly = DensePolynomial::from_coefficients_slice(
+            &self.cs.perm.compute_lookup_permutation_poly(
+                &domain,
+                &compressed_f_multiset.0,
+                &compressed_t_multiset.0,
+                &h_1.0,
+                &h_2.0,
+                delta,
+                epsilon,
+            ),
+        );
+
+        // TODO: Find strategy for blinding lookups
+        // Add blinder for lookup permutation poly
+        // z_2_poly = Self::add_blinder(&z_2_poly, n, 2);
```

Comment:
resolve

Reply:
We will implement plonkup blinding in a separate PR